app/Http/Controllers/FavoriteController.php

```diff
@@ -18,23 +19,34 @@ public function __construct(Favorite $favorites)
 
     public function index($user_id)
     {
-        $posts = $this->favorites->where('user_id', $user_id)->with('post');
-        return view('favorites.index')
-            ->with('user_id', $user_id)
-            ->with('posts', $posts);
+        $user2 = User::find(2); // IDが2のユーザーを取得
+        $posts = $user2->favorites()->get();
+        // return view('favorites.index')
+        //     ->with('user_id', $user_id);
+        return view('favorites.index', compact('posts', 'user_id'));
+
     }
 
-    public function store(Request $request)
-    {
-        $userId = $request->input('user_id');
-        $postId = $request->input('post_id');
-    
-        $this->favorites->user_id = $userId;
-        $this->favorites->post_id = $postId;
-        $this->favorites->save();
-    
+    public function store(Request $request, $post_id)
+{
+    $user2 = User::find(2); // IDが2のユーザーを取得
```

Comment:
don't set to specific id, i think you intedn to use` $post_id`

Reply:
Since our login function did NOT work so that Auth::user()->id was not available, we chose to put the specific id temporally. (The login function have been fixed on PR #112. )